plugins/source/slack/codegen/recipes/conversations.go

```diff
@@ -0,0 +1,94 @@
+package recipes
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/slack/resources/services/conversations/models"
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/slack-go/slack"
+)
+
+func ConversationResources() []*Resource {
+	resources := []*Resource{
+		{
+			SubService:  "conversations",
+			Description: "https://api.slack.com/methods/conversations.list",
```

Comment:
1. the API documentation should be the canonical source: it may include details and descriptions about the columns, limits and possible values not offered by SDK comments, as well as a number of examples
2. the field names in the Go SDK will be in camel case, not the same as the API
3. CloudQuery users are not necessarily familiar with Go, and shouldn't need to care whether our plugins are implemented in Go, Java, Python, or something else
4. for the above reasons we usually link to the API documentation in other plugins as well. There may be exceptions, but I think we should revisit that then.